build-parent/pom.xml

```diff
@@ -312,7 +312,7 @@
             </dependency>
 
             <dependency>
-                <groupId>org.subethamail</groupId>
+                <groupId>com.github.davidmoten</groupId>
```

Comment:
A compromise? No, it's a maintained fork of the original project which has been dead since forever (the last version was released 10 years ago).
I switch to it now so that the upgrade is easier as they provide both a version with javax and a version with jakarta and it's just a version bump once the upgrade is done.

Reply:
Ok, thanks. I guess I'm surprised we had a dependency to a long-dead project - and in in such a core component.

Reply:
oh, it's a test dependency. Nevermind :)